drivers/hwmon/nct7363.c

```diff
@@ -0,0 +1,396 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/*
+ * Copyright (c) 2023 Nuvoton Technology corporation.
```

Comment:
Reference the source where this driver came from, in the commit message?

Reply:
added in commit message of driver